### DISCUSSION TITLE
Skin editor causes softlock and/or crash

### DISCUSSION THREAD

**Discussion by eugenekahekili** [ppy/osu#14850] [Q&A]
**PC specs:**
CPU: Intel i7
Installed RAM: 16.0 GB
Operating System: Windows 11 Home
Laser version: 2021.916.0

**Description of issue**
When pressing "save changes" in the skin editor, the laser client crashes ~50% of the time. 100% of the time, when exiting a beatmap after using the skin editor, instead of going to the main menu, the game gets stuck on a solid picture of the map background. When the Alt+Home shortcut is used, the game, instead of going to the main menu, gets stuck on a solid image of the main menu background image. Game resumes normal function upon exiting and restarting, but skin changes are not saved.

**Comment by bdach** [comment]
Please attach game logs when this happens, as requested in issue templates.

**Comment by Wieku** [comment]
I can't reproduce it, for me the only way to cause a crash is clicking close window button when skin editor is opened:

https://user-images.githubusercontent.com/5307179/135181389-58dc1c82-67ff-40d1-89a9-d4cf8a9092f9.mp4

[runtime.log](https://github.com/ppy/osu/files/7247947/runtime.log)

**Reply by bdach** [reply]
This one is fixed by #14890.

**Reply by peppy** [reply]
Second issue will be fixed via #15204.
